lib/ansible_spec/load_ansible.rb

```diff
@@ -203,11 +203,14 @@ def self.load_dependencies(role)
       path = File.join("./", "roles", role, "meta", "main.yml")
 
       if File.exist?(path)
-        new_deps = YAML.load_file(path).fetch("dependencies", []).map { |h|
-          h["role"]
-        }
-        role_queue.concat(new_deps)
-        deps.concat(new_deps)
+        dependencies = YAML.load_file(path).fetch("dependencies", [])
```

Comment:
I think that Your ideas looks good.
But, your code is conflict because of merged another PR.
So, Could you resolve conflict?
And Could you add some test case?
(test case exist at `spec/dependency_spec.rb`)